### PR TITLE
[4.0][RTL] CSS logical properties initial

### PR DESCRIPTION
**Pull request by ciar4n**
Pull Request for Issue #28546 .

Also removed some redundant rules ( #28586 & #28585).

### Summary of Changes
Implements some logical properties to reduce on RTL CSS.

### Testing Instructions
Apply this patch and run `node build.js --compile-css` to update SCSS. Check RTL styling.

### CODE REVIEW

File: administrator/templates/atum/scss/blocks/_modals.scss [hunk 1]
```diff
@@ -66,14 +62,11 @@
     display: inline-block;
     width: 2.375rem;
     height: 100%;
-    margin: 0 16px 0 -22px;
+    margin: 0 16px;
```
Comment (C-Lodder) [inline administrator/templates/atum/scss/blocks/_modals.scss:65]: Can also be written as:
```
margin-block: 0;
margin-inline: -22px 16px;
```

**Review by wilsonge — APPROVED** [review]
LGTM on code review

**Comment by wilsonge** [comment]
@infograf768 can you test this please. If this works it will make maintaining RTL much easier.

File: administrator/templates/atum/scss/blocks/_modals.scss (hunk 1 above)
Comment (ciar4n) [reply administrator/templates/atum/scss/blocks/_modals.scss:65]: Thx @C-Lodder. Updated.

Comment (fabpacheco) [reply administrator/templates/atum/scss/blocks/_modals.scss:65]: margin-block and margin-inline are not supported in Safari
https://developer.mozilla.org/en-US/docs/Web/CSS/margin-block
https://developer.mozilla.org/en-US/docs/Web/CSS/margin-inline

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on eb0d8bb4f9855e035d8d6f208d7f5cde018e4221<br /><br />There are many changes and I'm not sure I tested all, but for what I tested it looks fine.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28589">issues.joomla.org/tracker/joomla-cms/28589</a>.</sub>

File: administrator/templates/atum/scss/blocks/_modals.scss (hunk 1 above)
Comment (ciar4n) [reply administrator/templates/atum/scss/blocks/_modals.scss:65]: Ah I missed that.. Thank you. Reverted.

**Comment by infograf768** [comment]
My tests were done with Firefox Macintosh

**Comment by ciar4n** [comment]
Thank you @infograf768

**Comment by wilsonge** [comment]
Think the toolbar cleanup you did conflicted here. If you fix the conflicts happy to get this in

**Comment by ciar4n** [comment]
Sorted. Once this is merged I will continue a cleanup across admin css.

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**